01_basic_stream/README.md

```diff
@@ -0,0 +1,135 @@
+# Simple stream
+
+This tutorial will get you started with the basic of using Anbox Cloud. By the end you will
+have a custom web page capable of streaming Android 10.
+
+## 1. Creating the application
+
+Create a directory and place a file named `manifest.yaml` in it.
+
+```shell
+$ mkdir android10
+$ touch android10/manifest.yaml
+```
+
+And copy the following in your manifest:
+
+```yaml
+# manifest.yaml
+name: my-android10-app
+instance-type: a2.3
+```
+
+Then use `amc` to create the application:
+
+```shell
+$ amc application create ./android10
+```
+
+`amc` will take a few minutes to prepare the application (apply security patches, optimize image size, run hooks, etc).
+You can monitor progress by running `amc application list`
+
+
+## 2. Building a web app to stream your Android applications
+
+Create a simple webpage containing an element with the ID `anbox-stream`. This is where the video stream will be displayed:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+</head>
+<body>
+    <div id="anbox-stream"></div>
+</body>
+```
+
+Then include the stream SDK:
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+    <script src="./path/to/anbox-stream-sdk.js"></script>
+</head>
+<body>
+    <div id="video-stream"></div>
+</body>
+```
+
+This will inject two classes in the `window` object.
+ * `window.AnboxStream` is the main object you'll interact with. It handles the actual WebRTC communication
+ * `window.AnboxStreamGatewayConnector` is a middleware that handles authentication for the Stream Gateway. If you want to later add your own authentication and application layer, you can implement your own connector without having to change the rest of the SDK
+
+----
+
+Let's start by preparing the connector
+
+```js
+const connector = new AnboxStreamGatewayConnector({
+    url : 'gateway.url.net',
+    authToken: 'YOUR_AUTH_TOKEN',
+    session: {
+        app: "com.foo.bar",
+    },
+});
+```
+
+> If you've deployd Anbox Cloud through the AWS marketplace image, you can replace `gateway.url.net` by `https://<ec2_ip>`.
+> On Juju based deployments, you would use the IP address of `anbox-stream-gateway-lb`
+
+The authentication token is obtained by running the following command
```

Comment:
If we have single-sign-on or HTTP basic auth in the service providing the web site we don't need this, right?

Reply:
We still need an auth token for the service to talk to the gateway

Reply:
Right, good point